sudoku/sudoku.py

```diff
@@ -0,0 +1,147 @@
+# solve sudoku puzzle
+
+
+def solve_board(input_board):
+    """Solve a sudoku puzzle
+
+    Args:
+        input_board (list of list[int]): list of lists with integers
+        representing the sudoku board
+
+    Returns:
+        list of list: solved sudoku board else prints unsolvable board
```

Comment:
Should return a Tuple (status of solution, and board).

This way someone using this as an API can actually make sense of the output.

No printing things.